src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/SkillsetsOperations.cs

```diff
@@ -0,0 +1,1131 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Search
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Models;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// SkillsetsOperations operations.
+    /// </summary>
+    internal partial class SkillsetsOperations : IServiceOperations<SearchServiceClient>, ISkillsetsOperations
+    {
+        /// <summary>
+        /// Initializes a new instance of the SkillsetsOperations class.
+        /// </summary>
+        /// <param name='client'>
+        /// Reference to the service client.
+        /// </param>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        internal SkillsetsOperations(SearchServiceClient client)
+        {
+            if (client == null)
+            {
+                throw new System.ArgumentNullException("client");
+            }
+            Client = client;
+        }
+
+        /// <summary>
+        /// Gets a reference to the SearchServiceClient
+        /// </summary>
+        public SearchServiceClient Client { get; private set; }
+
+        /// <summary>
+        /// Retrieves a Cognitive Search skillset in an Azure Search service.
+        /// <see href="https://docs.microsoft.com/azure/search/cognitive-search-tutorial-blob" />
+        /// </summary>
+        /// <param name='skillsetName'>
+        /// The name of the skillset to retrieve.
+        /// </param>
+        /// <param name='searchRequestOptions'>
+        /// Additional parameters for the operation
+        /// </param>
+        /// <param name='customHeaders'>
+        /// Headers that will be added to request.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        /// <exception cref="CloudException">
+        /// Thrown when the operation returned an invalid status code
+        /// </exception>
+        /// <exception cref="SerializationException">
+        /// Thrown when unable to deserialize the response
+        /// </exception>
+        /// <exception cref="ValidationException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null
+        /// </exception>
```

Comment:
Descriptions for ValidationException and ArgumentNullException are the same

Reply:
@vl8163264128 @jeji1101 You can fix it in [AutoRest](https://github.com/Azure/autorest) if you like. :)